README.md

```diff
@@ -1476,6 +1476,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Kuwar Kapur](https://github.com/kuwarkapur)
   - [Kyle Gonzales](https://github.com/kipppppp)
   - [Kyrios Heylel](https://github.com/lucky-chap)
+  - [Kennedy Musau](https://github.com/kenmusau)
```

Comment:
In order to stay alphabetically correct, your name should be between those two entries: 

  - [Ken Mwangi](https://github.com/kenmwangi)
  _- [Kennedy Musau](https://github.com/kenmusau)_
  - [Kenniku Tolato](https://github.com/knnku)

Reply:
Order has been resolved